.github/workflows/integration-test.yml

```diff
@@ -66,6 +66,21 @@ jobs:
           setup-script-name: setup
           admin-password: admin
 
+      # https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+      - name: Remove unnecessary files Linux
+        if: ${{ runner.os == 'Linux' }}
+        run: |
+          sudo rm -rf /usr/share/dotnet
+          sudo rm -rf "$AGENT_TOOLSDIRECTORY"
+
+      - name: Remove unnecessary files Windows
```

Comment:
@DarshitChanpura let's look into other ways to clear up space or adjust the watermark. I don't think its possible to tell the github runner to not come pre-installed with bloatware (it may not necessarily be bloatware, but we don't need it for our CI) which would be my preferred fix :/